jax/_src/lax/control_flow.py

```diff
@@ -2924,3 +2925,92 @@ def _cumulative_jvp_rule(primals, tangents, *, axis: int, reverse: bool,
 ad.primitive_jvps[cumprod_p] = partial(_cumulative_jvp_rule, combine_fn=lax.mul)
 ad.primitive_jvps[cummin_p] = partial(_cumulative_jvp_rule, combine_fn=lax.min)
 ad.primitive_jvps[cummax_p] = partial(_cumulative_jvp_rule, combine_fn=lax.max)
+
+
+def _dummy_remat_result(aval: core.AbstractValue):
+  """A result that will be discarded"""
+  if aval is core.AbstractToken:
+    return lax.create_token()
+  elif aval is core.abstract_unit:
+    return core.unit
+  else:
+    return lax.broadcast(np.array(0, dtype=aval.dtype), aval.shape)  # type: ignore
```

Comment:
lax.full_like ?

Reply:
For `lax.full_like` I need a value x to use the shape or dtype for. I could write `lax.full_like(None, 0, dtype=aval.dtype, shape=aval.shape)` but I think it is less clear that this is just a broadcast.